scripts/swarm-fix-route.sh

```diff
@@ -455,10 +472,10 @@ function main {
       s)
       echo "STEP 8:"
       echo "NAT EXCEPTIONS LIST:"
-      echo "---> show NAT UDP 514 exceptions in <<$INGRESS_NS>> namespace:"
+      echo "---> show NAT UDP/514 UDP/162 TCP/6514 exceptions in <<$INGRESS_NS>> namespace:"
```

Comment:
echo "---> show NAT UDP/514 UDP/162 TCP/514 TCP/6514 exceptions in <<$INGRESS_NS>> namespace:"